src/python/turicreate/test/test_sframe_s3.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2020 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+#
+from __future__ import print_function as _  # noqa
+from __future__ import division as _  # noqa
+from __future__ import absolute_import as _  # noqa
+from ..data_structures.sframe import SFrame
+
+import tempfile
+import os
+import math
+import shutil
+import pytest
+import boto3
+
+
+@pytest.mark.skipif(
```

Comment:
How slow are these test?

Reply:
depends on the network bandwidth. For my network, 2mb/s, it will take around 10 mins.